css/style.css

```diff
@@ -0,0 +1,30 @@
+.button-width {
+  width: 80px;
+  margin: 5px;
+}
+
+.question-size {
+  font-size: 30px;
+  font-family: 30px;
+}
+
+body {
+  background-image: url("../images/got-background.jpg");
```

Comment:
if you make the screen tall but not wide, your image will shrink to the width of the page and half the page will be white. You can solve it by finding a larger size image

![image](https://cloud.githubusercontent.com/assets/970858/10448362/f32dbb7a-71be-11e5-9d84-9c0a90be8ab6.png)